test/specs/scenarios/crossDevice.js

```diff
@@ -268,5 +268,21 @@ export const crossDeviceScenarios = async (lang) => {
       crossDeviceSubmit.clickOnSubmitVerificationButton()
       assert.isFalse(crossDeviceSubmit.submitVerificationButton.isEnabled(), 'Test Failed: Submit Verification button should be disabled')
     })
+
+    it('should complete cross device e2e flow with a US JWT', async () => {
+      driver.get(`${baseUrl}&region=US`)
+      welcome.primaryBtn().click()
+      documentSelector.passportIcon.click()
+      runThroughCrossDeviceFlow()
+      uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
+      selfieIntro.clickOnContinueButton()
+      camera.takeSelfie()
+      confirm.confirmBtn().click()
+      crossDeviceClientSuccess.verifyUIElements(copy)
```

Comment:
This doesn't actually test that cross device is calling a US endpoint. As part of another story we can detect the network request using something like [browsermob-proxy](https://github.com/lightbody/browsermob-proxy) to make sure that the request fired is calling the right endpoint.